core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
Nah, I'm not worried about it. We probably could have dropped Java 5 support in 2.4 given its EOL in May 2015.